ima_stub_service/installer.sh

```diff
@@ -9,16 +9,16 @@ if [[ $EUID -ne 0 ]]; then
    exit 1
 fi
 
-cp tpm_with_ima.sh /usr/local/bin/
-chmod 744 /usr/local/bin/tpm_with_ima.sh
-
 if [[ -n `systemctl 2>&1 > /dev/null` ]]; then
-	echo "No systemd on this system, starting TPM emulator service manually"
-	/usr/local/bin/tpm_with_ima.sh > /dev/null
-else
-	# create service
-	cp tpm_emulator.service /etc/systemd/system
-	chmod 664 /etc/systemd/system/tpm_emulator.service
-	systemctl enable tpm_emulator.service
-	systemctl start tpm_emulator.service
+	echo "No systemd on this system! Will not install emulator services"
+	exit 1
 fi
+
+# create services and socket
+for service in "tpm_emulator.service" "ima_emulator.service"; do
+  cp  ${service} /etc/systemd/system
+  chmod 644 /etc/systemd/system/${service}
+  systemctl enable ${service}
+done
+
+systemctl start ima_emulator.service
```

Comment:
I think there should be `systemctl daemon-reload` call before we start using new unit file.

Reply:
You are right. I'll make a PR that also cleans up some parts from the main installer.sh, that I forgot to remove.